src/frontend/src/lib/services/load-user-profile.services.ts

```diff
@@ -0,0 +1,54 @@
+import type { UserProfile } from '$declarations/backend/backend.did';
+import { createUserProfile, getUserProfile } from '$lib/api/backend.api';
+import { i18n } from '$lib/stores/i18n.store';
+import { userProfileStore } from '$lib/stores/settings.store';
+import { toastsError } from '$lib/stores/toasts.store';
+import { UserProfileNotFoundError } from '$lib/types/errors';
+import type { Identity } from '@dfinity/agent';
+import { isNullish } from '@dfinity/utils';
+import { get } from 'svelte/store';
+
+const getUncertifiedProfile = async ({
```

Comment:
can we use the prefix `unsafe` instead of `uncertified`?

Reply:
Done